script.js

```diff
@@ -0,0 +1,8 @@
+function sum()
+{
+    var x = parseInt(document.getElementById("varx").value);
+    var y = parseInt(document.getElementById("vary").value);
+    var result =x+y;
```

Comment:
For future reference, try to have consistent spacing in your code (for the sake of code readability). For example, here you have "var result =x+y" but it would be good practice to write is as "var result = x+y" or "var result = x + y" in the future, with equal spacing on each side of the equal sign.